packages/desktop/src/main/main.ts

```diff
@@ -30,6 +30,8 @@ const updaterInterval = 15 * 60_000
 export const isDev = process.env.NODE_ENV === 'development'
 export const isE2Etest = process.env.E2E_TEST === 'true'
 
+const SOCKET_IO_TOKEN = generateJWT()
```

Comment:
Why not just use a random number as a token with basic auth? What benefit does a JWT provide here?